### PR TITLE
Ef/sec bot demo

### PR DESCRIPTION
**Pull request by ecfidler**
The added data and commands for the first terminal demo!!!!! woo yeah!!!!

### CODE REVIEW

File: content/terminalCommands.json [hunk 1]
```diff
@@ -9,6 +9,62 @@
...
+    "prettyName": "DMs",
+    "path": "/root/whyd/SecBot",
+    "functionName": "SecBotDMsFunction"
+  },
+  "scores": {
+    "prettyName": "Scores",
+    "path": "/root/whyd/SecBot",
+    "functionName": "SecBotScores"
+  },
+  "score highs": {
```
Comment (benrucker) [inline content/terminalCommands.json:50]: are the `id`s of any of these values ever used? e.g. `"score highs"`

Comment (benrucker) [reply content/terminalCommands.json:50]: also, are these sorted in any way? or is it just ad-hoc?

File: components/whyd/2022/Terminal/UserInputArea.vue [hunk 2]
```diff
@@ -299,6 +356,230 @@ export default {
...
+    },
+    MetaSecBotDevelopment() {
+      // describe how the scoring worked? including the initial method and how it was changed to be wackier after a few weeks while ben was drunk? (cheeto cheeto)
+      // include how scores loop around from 1600
+      // at 7am on april 1st Ethan and Ben met up to make sure that it worked, and it didnt so we needed to manually activate it.
+      // creation of Dennis
+      // CUMPEG
+      // format all of these like they are chat logs between Ben and Ethan?
+      // have some corrupted / jumbled text
+      // folder title: subpoena documents, backups,
```
Comment (benrucker) [inline components/whyd/2022/Terminal/UserInputArea.vue]: I feel like this file should be re-composed into smaller chunks ASAP

Comment (benrucker) [reply components/whyd/2022/Terminal/UserInputArea.vue]: a low effort possibility would be to extract the "command functions" out into their own file and import it. maybe this file also deserves a rename if its scope has changed

Comment (benrucker) [reply components/whyd/2022/Terminal/UserInputArea.vue]: tbf tho this is a problem on the `Chat` side as well - `Text.vue` includes mappings of textual representation of data to the actual data itself. however, Text.vue doesn't actually contain or instantiate any data/text - it just maps from a key to a value.

File: content/terminalCommands.json (hunk 1 above)
Comment (benrucker) [reply content/terminalCommands.json:50]: i don't think it necessarily needs to change, my eye was simply caught by the fact that there are 3 unique keys for each element here

**Comment by ecfidler** [comment]
It’s what ppl can type in to see the command again when the button
disappears, shows up in LS list.

**Comment by benrucker** [comment]
> It’s what ppl can type in to see the command again when the button disappears, shows up in LS list.

is prettyname what shows up on the buttons?

**Pull request merged**